spec/lib/ecco/row_event_listener_spec.rb

```diff
@@ -53,6 +53,9 @@
 
         context "UPDATE_ROWS" do
           let(:row_event_type) { EventType::UPDATE_ROWS }
+          let(:update_entry) { Java::JavaUtil::Map.entry([], []) }
+          let(:rows) { Java::JavaUtil::ArrayList.new(Array.new(1) { update_entry }) }
```

Comment:
this looks strange, a mixture between Java and Ruby objects? `Array` being Ruby.

Reply:
Didn't think too much about wether it was a Java or a Ruby array at this stage, but yea I agree, should probably be a Java construct throughout the reflect the real world better.

Reply:
Fixed here: 186eecbac3b2e4631996eb515e627839a6c0e8b8